WcaOnRails/app/helpers/competitions_helper.rb

```diff
@@ -13,4 +13,38 @@ def competition_message_for_user(competition, user)
                 end
     messages.join(' ')
   end
+
+  def announced_class(days_announced)
+    if days_announced <= 21
+      "alert-danger"
+    elsif days_announced <= 28
```

Comment:
Is this the same 28 as our [NEARBY_DAYS_DANGER](https://github.com/cubing/worldcubeassociation.org/blob/d7ded260b4e029fba58d1255f37a6f2be04411cc/WcaOnRails/app/models/competition.rb#L79)?

Pulling these numbers out to named constants wouldn't make me ask these questions =)